istioctl/cmd/version.go

```diff
@@ -64,6 +66,16 @@ func getRemoteInfo() (*istioVersion.MeshInfo, error) {
 	return kubeClient.GetIstioVersions(istioNamespace)
 }
 
+func getRemoteInfoWrapper(pc **cobra.Command) func() (*istioVersion.MeshInfo, error) {
```

Comment:
Why introduce a wrapper? The method is only used in this one place, it might as well implement proper error handling directly